specs/merkle-tree.md

```diff
@@ -122,16 +122,14 @@ To ensure that `APPROVE` and `CANCEL` leaves always happen first on a given chai
 ### 8. Each sequence of EXECUTE leaves with the same `chainId` must have the index of the first leaf be `1` and the index of the following leaves sequentially increment by `1`
 To ensure that all the `EXECUTE` leaves are executed in the correct order, they must have an `index` field that starts at `1` on each chain and sequentially increments by `1`.
 
-### 9. Merkle tree leaves must be ordered in the tree by the leaf's `index` and `chainId` fields ascending
-Sphinx Merkle tree leaves use an explicit `index` field in the leaf rather than an implicit index determined by the leaf's position in the tree. This is because the Sphinx Merkle tree is intended to be executed across multiple networks, and not all leaves will be executed on all networks. We impose the restriction that tree leaves must be ordered by their `index` and `chainId` fields ascending starting at the leftmost position within the tree.
+### 9. Merkle tree leaves must be ordered in the tree by the leaf's hash descending
+To allow for Sphinx Merkle trees to be reliably reproduced by off-chain tooling, we impose the restriction that Sphinx Merkle trees must have their leaves ordered within the tree by their hash descending.
 
-So, for example, if we intended to run a single transaction (one `EXECUTE` leaf) on Goerli (5), OP Goerli (420), and Polygon Mumbai (80001), then the leaves are expected to be ordered like so:
-1. `APPROVE` Leaf on Goerli (Chain ID 5)
-2. `EXECUTE` Leaf on Goerli
-3. `APPROVE` Leaf on OP Goerli (Chain ID 420)
-4. `EXECUTE` Leaf on OP Goerli
-5. `APPROVE` Leaf on Polyon Mumbai (Chain ID 80001)
-6. `EXECUTE` Leaf on Polygon Mumbai
```

Comment:
Why remove the example? If anything, it may be more helpful now because the order of the leaves is a little more complicated

Reply:
I was thinking about including this but decided against it. Since the leaves are ordered according to their hashes and most people can't intuitively tell what the correct order of the hashes should be, the example just looks like it's randomly ordered. So it doesn't really add anything wrt to helping the reader understand the ordering mechanism. I think it's better to focus on directing their attention to an example of the formula used to calculate the hash.